packages/contracts/src/abis/rfox.ts

```diff
@@ -1,6 +1,4 @@
-import type { InterfaceAbi } from 'ethers'
-
-export const RFOX_ABI: InterfaceAbi = [
```

Comment:
I'd rather we didn't untype these - goes for all the similar diff across all ABI files.

Reply:
We need to remove the type here as it replaces the structural information contained in the ABI with a generic ABI type which negates the benefit of using an abi here